Test_OrderedSet.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from containers.orderedset import OrderedSet
+
+class TestOrderedSetMethods(unittest.TestMethod):
+    def test_constructor(self):
+        try:
+            orderedset_a = OrderedSet()
+            orderedset_b = OrderedSet([1,2,3,4,5])
+            print('TEST CONSTRUCTOR: PASS')
+        except:
+            print('TEST CONSTRUCTOR: FAIL')
+
+    def test_size(self):
+        try:
+            orderedset = OrderedSet()
+            orderedset.size()
```

Comment:
because size returns something i think the proper thing to do is to test the return value (for clarity, even though you do test size later)

also by testing the return value you test the existence of the function as well